types/chayns/index.d.ts

```diff
@@ -0,0 +1,938 @@
+// Type definitions for chayns 0.14
+// Project: https://gitlab.tobit.com/chayns-api/chayns-typings
+// Definitions by: Henning Kuehl <https://github.com/HenningKuehl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+/**
+ * Definition file for chayns v3.1
```

Comment:
The version in the header is what ends up as the version of @types package. the policy is that this version matches the major.minor of the package it defines, any updates to the definition file will have a build number pump.